src/components/container/RecentDailyDataBarChart/RecentDailyDataBarChart.css

```diff
@@ -32,7 +32,7 @@
     width: 120px;
     height: 20px;
     line-height: 20px;
-    left: calc(50% - 76px);
+    left: calc(50% - 68px);
```

Comment:
Flagging this as it looks unrelated to changes. 
Please disregard if it was intentional.

Reply:
Thanks @reneefromhold.  Yes, this was intentional.  I noticed that the no-data label was a tad bit off center.  It should have been at the midpoint (50%), minus half the width (60), minus the left padding (8).